Cargo.toml

```diff
@@ -52,34 +55,34 @@ indoc = "2.0.0"
 
 # Dependencies from nextjs, should keep the same rev!
 ## Dependencies for common cache and task. See https://github.com/umijs/next.js/blob/canary/turbopack/crates/turbopack/architecture.md
-turbo-tasks = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-fs = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-fs", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-env = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-env", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-backend = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-backend", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-bytes = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-bytes", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-fetch = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-fetch", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-hash = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-hash", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-build = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-build", branch = "utoo-patch-v15.3.2" }
-turbo-rcstr = { git = "https://github.com/umijs/next.js.git", package = "turbo-rcstr", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-memory = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-memory", branch = "utoo-patch-v15.3.2" }
-turbo-tasks-malloc = { git = "https://github.com/umijs/next.js.git", package = "turbo-tasks-malloc", branch = "utoo-patch-v15.3.2", default-features = false }
-turbopack-trace-utils = { git = "https://github.com/umijs/next.js.git", package = "turbopack-trace-utils", branch = "utoo-patch-v15.3.2" }
-turbopack-trace-server = { git = "https://github.com/umijs/next.js.git", package = "turbopack-trace-server", branch = "utoo-patch-v15.3.2" }
+turbo-tasks = { path="../umi-next/turbopack/crates/turbo-tasks" }
```

Comment:
No local dependencies.

Reply:
i will update the dependencies here after https://github.com/umijs/next.js/pull/19 merged

Reply:
resolved